From f70d3c2961d45cf8e63b32832985184589f25e60 Mon Sep 17 00:00:00 2001
From: Dylan Van Assche <me@dylanvanassche.be>
Date: Sun, 19 May 2024 18:21:40 +0200
Subject: [PATCH 03/11] fbd-dev-led: Scan for RGB color as well

Multicolor LEDs have the color 'rgb', we should also scan it
as they are also a FbdDevLed.
---
 src/fbd-dev-led.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fbd-dev-led.c b/src/fbd-dev-led.c
index 8c48e67..f577229 100644
--- a/src/fbd-dev-led.c
+++ b/src/fbd-dev-led.c
@@ -63,7 +63,7 @@ fbd_dev_led_probe_default (FbdDevLed *led, GError **error)
   /* We don't know anything about diffusors that can combine different
      color LEDSs so go with fixed colors until the kernel gives us
      enough information */
-  for (int i = 0; i < FBD_FEEDBACK_LED_COLOR_RGB; i++) {
+  for (int i = 0; i <= FBD_FEEDBACK_LED_COLOR_LAST; i++) {
     g_autofree char *color = NULL;
     g_autofree char *enum_name = NULL;
     gchar *c;
-- 
2.45.1

